sdk/core/core-rest-pipeline/package.json

```diff
@@ -14,11 +14,11 @@
     "./dist-esm/src/util/url.js": "./dist-esm/src/util/url.browser.js",
     "./dist-esm/src/util/userAgentPlatform.js": "./dist-esm/src/util/userAgentPlatform.browser.js"
   },
-  "types": "types/latest/core-rest-pipeline.d.ts",
+  "types": "core-rest-pipeline.shims.d.ts",
   "typesVersions": {
     "<3.6": {
-      "types/latest/src/*": [
-        "types/3.1/src/*"
+      "types/latest/*": [
```

Comment:
TypeScript docs say this field should work "just like path mappings", so I think this works without providing another shim, but need to give it a quick test.

Reply:
Turns out this doesnt work.

Reply:
Turns out this doesn't work.